src/main/java/mate/academy/internetshop/controller/LogoutController.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.internetshop.controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.*;
+import java.io.IOException;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        HttpSession session = req.getSession(true);
```

Comment:
we can simplify:
HttpSession session = req.getSession(true);
        session.setAttribute("userId", "");
        session.invalidate();
-->
req.getSession().invalidate()